lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -299,7 +299,13 @@
         type: bool
         default: no
         version_added: "2.5"
-
```

Comment:
Please keep this empty line, and add another after the new option.

Reply:
Thanks!